cypress/e2e/labels.test.cy.js

```diff
@@ -0,0 +1,42 @@
+
+describe("test labels", () => {
+  it("passes", () => {
+    cy.visit("http://127.0.0.1:5500/challenges.html");
+    cy.get(".filterButton").click();
+
+    cy.get("#linux")
+      .should("have.text", "Linux")
+      .click()
+      .should("have.class", "active")
+      .click()
+      .should("not.have.class", "active");
+
+    cy.get("#cryptography")
+      .should("have.text", "Cryptography")
+      .click()
+      .should("have.class", "active")
+      .click()
+      .should("not.have.class", "active");
```

Comment:
This checks that toggling classes works on `#cryptography`. Is there any reason to believe it wouldn't, if it does already work for `#linux`?